pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -242,6 +242,22 @@ func CreateService(name string, service kobject.ServiceConfig, objects []runtime
 	servicePorts := ConfigServicePorts(name, service)
 	svc.Spec.Ports = servicePorts
 
+	// Configure service types
+	for key, value := range service.Annotations {
+		if key == "kompose.service.type" && (value == "NodePort" || value == "nodeport") {
+			svc.Spec.Type = "NodePort"
+			delete(service.Annotations, "kompose.service.type")
+		} else if key == "kompose.service.type" && (value == "ClusterIP" || value == "clusterip") {
+			svc.Spec.Type = "ClusterIP"
+			delete(service.Annotations, "kompose.service.type")
+		} else if key == "kompose.service.type" && (value == "LoadBalancer" || value == "loadbalancer") {
+			svc.Spec.Type = "LoadBalancer"
+			delete(service.Annotations, "kompose.service.type")
+		} else {
+			logrus.Fatalf("Unknown Service Type %s", value)
```

Comment:
If there is any other label than `kompose.service.type` it fails on this.

It would be better to break this into two nested `ifs` or `switch` in `if`.
In first level you check if there is `kompose.service.type` key, and inside that you do `else if` or `switch` on values.